fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -359,8 +359,8 @@ bool verifyTenantPrefix(ProxyCommitData* const commitData, const CommitTransacti
 }
 
 ACTOR Future<Void> commitBatcher(ProxyCommitData* commitData,
+                                 CommitProxyInterface cpi,
```

Comment:
Now that you don't need anything from CommitProxyInterface, you could go back to the original signature.